pkg/config/config.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/aws/eks-anywhere/pkg/logger"
+)
+
+const (
+	EksaGitPasswordTokenEnv   = "EKSA_GIT_PASSWORD"
+	EksaGitPrivateKeyTokenEnv = "EKSA_GIT_PRIVATE_KEY"
+)
+
+type CliConfig struct {
+	GitPassword       string
+	GitPrivateKeyFile string
+}
+
+func ReadEnvVar(EksaEnvVar string) (string, error) {
```

Comment:
What do you think about 
```suggestion
func ReadExpectedEnvVar(EksaEnvVar string) (string, error) {
```